internal/client/slack.go

```diff
@@ -120,15 +120,29 @@ func (sc slackClient) PostEvent(ctx context.Context, event *AlertEventTrigger) (
 		}
 	}()
 
-	// 3. read and unmarshal response
+	// 3.a. read and validate response
 	bytes, err := io.ReadAll(resp.Body)
 	if err != nil {
 		return nil, err
 	}
 
+	// 3.b. validate status code
+	if resp.StatusCode != http.StatusOK {
+		return nil, fmt.Errorf("slack API returned bad status code %d", resp.StatusCode)
+	}
+
+	// 3.c. validate response body
+	if string(bytes) == "ok" {
```

Comment:
[nit] Using a const for slack Ok status might be better here

Reply:
[6e20b7d](https://github.com/base-org/pessimism/pull/173/commits/6e20b7d993c03f8d627fa675659949f6b701b67c)